tests/WebToPay/Config/EnvReaderTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+use PHPUnit\Framework\TestCase;
+
+class WebToPay_EnvReaderTest extends TestCase
+{
+    /**
+     * @dataProvider getAsArrayData
+     */
+    public function testGetAsString(string $key, ?string $expected, ?string $actual): void
+    {
+        $this->assertEquals($expected, $actual);
+    }
+
+    public function getAsArrayData(): iterable
+    {
+        $envReader = new WebToPay_EnvReader();
```

Comment:
A bit strange to see SUT here, in data provider. Usually, data provider is just to provide various inputs to the test. And SUT can reside in setUp or test itself.

Also, I'd put manipulation with $_ENV and printenv to the setUp / tearDown.

Reply:
Done, changed the test